pythonx/UltiSnips/snippet_manager.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, expand_trigger, forward_trigger, backward_trigger):
         enable_snipmate = True
         if _vim.eval("exists('g:UltiSnipsEnableSnipMate')") == "1":
             enable_snipmate = _vim.eval("g:UltiSnipsEnableSnipMate")
-        if enable_snipmate:
+        if enable_snipmate == "1":
```

Comment:
why not use `int(enable_snipmate)`, would that not handle strings and integers in the settings?